lte/gateway/python/integ_tests/s1aptests/test_agw_offload_idle_active_ue.py

```diff
@@ -72,16 +73,20 @@ def test_agw_offload_idle_active_ue(self):
         assert self._ha_util.offload_agw(
             "".join(["IMSI"] + [str(i) for i in req.imsi]),
             enb_list[0][0],
+            EnbOffloadType.ANY_CONNECTED,
         )
```

Comment:
Out of curiosity: how does this change impact the test?
The default on master for this parameter in `offload_agw()` is 0 -> `EnbOffloadType.ALL`. I would have guessed an `ALL` to be more stable (=less prone to errors) than an `ANY_CONNECTED`/`ANY_IDLE`.

Reply:
In general yes, but this is more specific of what the test is actually trying to do, and it makes it more readable to what is happening. It should not impact the test at all (so I can also just remove it again if it seems unnecessary)